src/auro-alaska.js

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable one-var */
+// Copyright (c) 2020 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+import { html, css } from "lit-element";
+import { classMap } from 'lit-html/directives/class-map';
+import BaseIcon from "./baseIcon";
+
+import as400 from '@alaskaairux/icons/dist/restricted/AS-400_es6.js';
+import as300 from '@alaskaairux/icons/dist/restricted/AS-300_es6.js';
+import as200 from '@alaskaairux/icons/dist/restricted/AS-200_es6.js';
+import as100 from '@alaskaairux/icons/dist/restricted/AS-100_es6.js';
+import asTag400 from '@alaskaairux/icons/dist/restricted/AS-tagline-400_es6.js';
+import asTag300 from '@alaskaairux/icons/dist/restricted/AS-tagline-300_es6.js';
+import asTag200 from '@alaskaairux/icons/dist/restricted/AS-tagline-200_es6.js';
+import asTag100 from '@alaskaairux/icons/dist/restricted/AS-tagline-100_es6.js';
+
+// Import touch detection lib
+// import "focus-visible/dist/focus-visible.min.js";
+import styleCss from "./alaskaStyle-css.js";
+
+// See https://git.io/JJ6SJ for "How to document your components using JSDoc"
+/**
+ * auro-alaska provides users a way to use the Auro Icons by simply passing in the category and name.
+ *
+ * @attr {Boolean} alaska - Set value for default alaska airlines logo
+ * @attr {Boolean} alaskaOfficial - Set value for alaska airlines logo with official tagline
+ * @slot - Hidden from visibility, used for a11y if icon description is needed
+ */
+
+// build the component class
+class AuroAlaska extends BaseIcon {
+  constructor() {
+    super();
+
+    this.privateDefaults();
+  }
+
+  /**
+   * @private internal defaults
+   * @returns {void}
+   */
+  privateDefaults() {
+    this.uri = 'https://unpkg.com/@alaskaairux/icons@latest/dist';
+    this.zero = 0;
+    this.no = 1;
+    this.sm = 107;
+    this.md = 191;
+    this.lg = 527;
+  }
+
+  // function to define props used within the scope of this component
+  static get properties() {
+    return {
+      ...super.properties,
+      alaska: {
+        type: Boolean,
+        reflect: true
+      },
+      alaskaOfficial: {
+        type: Boolean,
+        reflect: true
+      },
+    };
+  }
+
+  /**
+   * @private async function to decide which logo to use
+   * @param {number} iconWidth size of parent icon is in
+   * @returns {object} SVG to be appended to DOM
+   */
+  alaskaLogoDef(iconWidth) {
+    switch (this.alaska) {
+
+      case iconWidth <= this.sm:
+        this.svg = new DOMParser().parseFromString(as100.svg, 'text/html').body.firstChild;
+        break;
+      case iconWidth >= this.sm + this.no && iconWidth <= this.md:
```

Comment:
Are we using this.no to get around an ESLint rule? I think this is much more readable as `this.sm + 1`

Reply:
Magic numbers man ... magic freaking numbers.

Reply:
I think you can get rid of `this.no` altogether if you use > instead of >=

```suggestion
      case iconWidth > this.sm && iconWidth <= this.md:
```

Reply:
Yup. I struggle with being overly literal at times. :+1: